signature-specification.md

```diff
@@ -92,28 +92,27 @@ Unless explicitly specified as OPTIONAL, all fields are required. Also, there sh
 
 **JWSPayload a.k.a. Claims**:
 Notary v2 is using one private claim(`notary`) and two public claims(`iat` and `exp`). An example of the claim is described below
+
 ```json
 {
-   "notary": {  
-       "subject": {
-           "mediaType": "application/vnd.oci.image.manifest.v1+json",
-           "digest": "sha256:73c803930ea3ba1e54bc25c2bdc53edd0284c62ed651fe7b00369da519a3c333",
-           "size": 16724,
-           "annotations": {
-              "key1": "value1",
-              "key2": "value2",
-               ...
-           }
-       }
+   "notary": {
```

Comment:
This sounds more like a subject rather than the `notary`

Reply:
+1, the data stored is subjectDescriptor.